x-pack/plugins/cloud_security_posture/server/routes/stats/stats.test.ts

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import {
```

Comment:
The '[functional test runner](https://www.elastic.co/guide/en/kibana/master/development-tests.html#development-functional-tests)' can be used for integration tests that call an actual ES cluster. That might be worth considering when testing APIs that rely heavily on ES.

Reply:
We plan on leveraging these tests in the near future :)